web/app/view/NotificationsController.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2016 Anton Tananaev (anton.tananaev@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Ext.define('Traccar.view.NotificationsController', {
+    extend: 'Ext.app.ViewController',
+    alias: 'controller.notificationsController',
+
+    init: function () {
+        this.userId = this.getView().user.getData().id;
+        this.getView().getStore().load({
+            scope: this,
+            callback: function (records, operation, success) {
+                var notificationsStore = Ext.create('Traccar.store.Notifications');
+                notificationsStore.load({
+                    params: {
+                        userId: this.userId
+                    },
+                    scope: this,
+                    callback: function (records, operation, success) {
+                        var i, index, attributes, storeRecord;
+                        if (success) {
+                            for (i = 0; i < records.length; i++) {
+                                index = this.getView().getStore().find('type', records[i].getData().type);
+                                attributes = records[i].getData().attributes;
+                                storeRecord = this.getView().getStore().getAt(index);
+                                if (typeof attributes.web != 'undefined') {
```

Comment:
It will ruin our idea about easy extendable notification object.
I can try to do it via `renderers`. it will look more cleaner, but still we will have to construct request from plane object.